blueprints/app/files/.eslintrc.js

```diff
@@ -48,6 +48,17 @@ module.exports = {
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
         // add your custom rules and overrides for node files here
       })<% } %>
-    }
+    }<% if (blueprint !== 'app') { %>,
+
+    // dummy node files
+    {
+      files: [
+        'ember-cli-build.js',
+        'tests/dummy/config/**/*.js'
+      ],
+      rules: {
+        'node/no-unpublished-require': 'off'
```

Comment:
This rule already does not run on files that are not published to `npm`, disabling the rule is probably not the right answer. We should instead fix our `.npmignore`s (and the rule is doing its job)...

Reply:
You're right. Correcting the .npmignore made everything better.